packages/pods-core/src/builtin/AirtablePod.ts

```diff
@@ -0,0 +1,187 @@
+
+export type AirtableExportConfig = ExportPodConfig &
+  AirtableExportPodCustomOpts;
+
+type AirtableExportPodProcessProps = AirtableExportPodCustomOpts & {
+  filteredNotes: NoteProps[];
+  checkpoint: string;
+};
+
+export class AirtableExportPod extends ExportPod<AirtableExportConfig> {
+  static id: string = ID;
+  static description: string = "export notes to airtable";
+
+  get config() {
+    return super.config.concat([
+      {
+        key: "tableName",
+        description: "Name of the airtable",
+        type: "string",
+        required: true,
+      },
+      {
+        key: "srcHierarchy",
+        description: "The src .md file from where to start the sync",
+        type: "string",
+        required: true,
+      },
+      {
+        key: "apiKey",
+        description: "Api key for airtable",
+        type: "string",
+        required: true,
+      },
+      {
+        key: "baseId",
+        description: " base Id of airtable base",
+        type: "string",
+        required: true,
+      },
+      {
+        key: "srcFieldMapping",
+        description:
+          "mapping of airtable fields with the note eg: {Created On: created, Notes: body}",
+        type: "object",
+        required: true,
+      },
+    ]);
+  }
+
+  //filters the note's property as per srcFieldMapping provided
+  notesToSrcFieldMap(notes: NoteProps[], srcFieldMapping: any) {
+    const data: any[] = notes.map((note) => {
+      let fields = {};
+      for (const [key, value] of Object.entries(srcFieldMapping)) {
+        fields = {
+          ...fields,
+          [key]: _.get(note, `${value}`).toString(),
+        };
+      }
+      return { fields };
+    });
+    return data;
+  }
+
+  async processNote(opts: AirtableExportPodProcessProps) {
+    const {
+      filteredNotes,
+      apiKey,
+      baseId,
+      tableName,
+      checkpoint,
+      srcFieldMapping,
+    } = opts;
+
+    const headers = {
+      Authorization: `Bearer ${apiKey}`,
+      "Content-Type": "application/json",
+    };
+
+    const records = this.notesToSrcFieldMap(filteredNotes, srcFieldMapping);
+    const chunks = _.chunk(records, 10);
+
+    // rate limiter method
+    const sendRequest = async () => {
+      let total: number = 0;
+      chunks.forEach(async (record) => {
+        await limiter.removeTokens(1);
+        const data = JSON.stringify({ records: record });
+        try {
+          const result = await axios.post(
+            `https://api.airtable.com/v0/${baseId}/${tableName}`,
+            data,
+            { headers: headers }
+          );
+          total = total + result.data.records.length;
+          if (total === filteredNotes.length) {
+            const timestamp = filteredNotes[filteredNotes.length - 1].created;
+            fs.writeFileSync(checkpoint, timestamp.toString(), {
+              encoding: "utf8",
+            });
+          }
+        } catch (error) {
+          console.log("failed to export all the notes. Error: ", error);
```

Comment:
use the logger that comes with the pod for logging. see https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.setup.md#L66:L66